docs/installing.md

```diff
@@ -60,13 +60,15 @@ Create a new CloudFormation stack using the [launch_demo](../bin/launch_demo)
 script.
 
 ```console
-$ ./bin/launch_demo
+$ ./bin/bootstrap
```

Comment:
This a couple areas in this branch where we reference `launch_demo`:

```
$ git grep 'launch_demo'
docs/installing.md:59:Create a new CloudFormation stack using the [launch_demo](../bin/launch_demo)
docs/using.md:10:Going forward from the [installing](./installing.md) guide, the first thing we'll need to do is tell the empire client where it can find the empire API. The *launch_demo* command should have printed that out for you, so make sure you've set it in your environment like so:
```